app/routes/route_helpers.py

```diff
@@ -0,0 +1,17 @@
+from flask import make_response, jsonify
```

Comment:
Great work encapsulating this functionality in helper functions and putting them in their own file to use through the code

Reply:
Consider how you can use `abort` to use of these functions.